src/components/paradox/results/personality/career/inline-jobs/auto.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from "prop-types";
+import {useEffect, useRef, useState} from "react";
+import InlineJobs from "./index";
+import style from "./style.scss";
+
+function InlineJobsAuto({jobs, jobSource, translate}) {
+  const ref = useRef(null);
+  const [count, setCount] = useState(0);
+
+  /** Adjusts number of jobs based on screen size */
+  const configureCount = () => {
+    const INLINE_JOB_WIDTH = 180;
+    const NEXT_BUTTON_SIZE = 44;
+    const {offsetLeft: OFFSET_LEFT, offsetWidth: WIDTH} = ref.current;
+    const {offsetWidth: PARENT_WIDTH} = ref.current.offsetParent;
+    const WIDTH_ESTIMATE = PARENT_WIDTH - OFFSET_LEFT;
+    const DIFFERENCE = WIDTH_ESTIMATE - WIDTH;
+
+    const VALUE = DIFFERENCE <= 0 ? Math.floor(PARENT_WIDTH - OFFSET_LEFT) : WIDTH;
+
+    const newCount = Math.floor((VALUE - (NEXT_BUTTON_SIZE * 2)) / INLINE_JOB_WIDTH);
+    setCount(newCount);
+  };
+
+  useEffect(() => {
+    configureCount();
+  }, []);
+
+  window.addEventListener("resize", configureCount);
```

Comment:
This needs to be inside a useEffect. I think the `configureCount` function could also be moved within the above useEffect. Then it would be something like

```
useEffect(() => {
  const configureCount = ...

  configureCount();
  
  window.addEventListener("resize", configureCount);
  
  return () =>   window.removeEventListener("resize", configureCount);
}, []);
```

Reply:
Yeah since the only time its used is in the useEffect. Now moved into useEffect!